kong/plugins/ldap-auth/access.lua

```diff
@@ -84,15 +90,29 @@ local function load_credential(given_username, given_password, conf)
   return {username = given_username, password = given_password}
 end
 
+
+local function cache_key(conf, username)
+  if not ldap_config_cache[conf] then
+    ldap_config_cache[conf] = md5(fmt("%s:%u:%s:%s:%u",
+      conf.ldap_host,
+      conf.ldap_port,
+      conf.base_dn,
+      conf.attribute,
+      conf.cache_ttl
+    ))
```

Comment:
we should remove case sensitivity from the cache-key for those elements that are not case sensitive (eg. hostname, haven't looked into the ldap specifc ones).

Reply:
LDAP specific ones are **usually** `case-insensitive`, but you may possibly configure them to be `case-sensitive`.

Reply:
then the safe way would be to keep them case-sensitive, so that only leaves the hostname to update.

Reply:
http://ldapwiki.com/wiki/Distinguished%20Name%20Case%20Sensitivity

(I don't have a courage to make other than `host` case-insensitive).

Reply:
aren't the host and port properties only used to connect to the ldap server?